docs/features/configuration.md

```diff
@@ -1,9 +1,10 @@
-# Custom properties
+# Custom configuration
```

Comment:
"Customizing ryuk resource reaper" section is wrongly rendered:
https://deploy-preview-1150--testcontainers.netlify.com/features/configuration/#customizing-ryuk-resource-reaper

Reply:
Fixing